libcaf_net/src/net/middleman.cpp

```diff
@@ -57,6 +59,15 @@ void middleman::init_global_meta_objects() {
   // nop
 }
 
+actor_system::global_state_guard middleman::init_host_system() {
+  this_host::startup();
+  ssl::startup();
+  return actor_system::global_state_guard{+[] {
```

Comment:
What does the `+` do?

Reply:
Turn the lambda into a function pointer. https://stackoverflow.com/questions/18889028/a-positive-lambda-what-sorcery-is-this

Reply:
I looked on [cppreference](https://en.cppreference.com/w/cpp/language/lambda) and couldn't find it. Good to know!